containers/ecr-viewer/src/app/utils/evaluate/fhir-paths.ts

```diff
@@ -314,6 +320,30 @@ const _fhirPathMappings: { [K in FhirPathKeys]: Omit<FhirPath<K>, "name"> } = {
     type: "ValueX",
     path: "Bundle.entry.resource.where(resourceType = 'Patient').maritalStatus",
   },
+  patientNationality: {
+    type: "string",
+    path: "Bundle.entry.resource.where(resourceType = 'Observation').where(code.coding.where(code = '186034007' and system = 'http://snomed.info/sct')).valueCodeableConcept.coding.display",
+  },
+  patientCountryResidence: {
+    type: "string",
+    path: "Bundle.entry.resource.where(resourceType = 'Observation').where(code.coding.where(code = '77983-5' and system = 'http://loinc.org')).valueCodeableConcept.coding.display",
+  },
+  patientDisabilityStatus: {
+    type: "Observation",
+    path: "Bundle.entry.resource.where(resourceType = 'Observation').where(meta.profile = 'http://hl7.org/fhir/us/ecr/StructureDefinition/us-ph-disability-status')",
+  },
+  disabilityStatusQuestion: {
+    type: "string",
+    path: "code.coding.display",
+  },
+  disabilityStatusValue: {
+    type: "boolean",
+    path: "valueBoolean",
+  },
+  disabilityStatusDate: {
+    type: "string",
+    path: "effectiveDateTime",
+  },
```

Comment:
```suggestion
  disabilityStatusDate: {
    type: "TimeX",
    path: "effective",
  },
```

this is the new stuff around more generically handling dates/times